crates/chain/src/spk_iter.rs

```diff
@@ -151,12 +150,25 @@ mod test {
     #[allow(clippy::iter_nth_zero)]
     fn test_spkiterator_wildcard() {
         let (_, external_desc, _) = init_txout_index();
-        let external_spk_0 = external_desc.at_derivation_index(0).script_pubkey();
-        let external_spk_16 = external_desc.at_derivation_index(16).script_pubkey();
-        let external_spk_20 = external_desc.at_derivation_index(20).script_pubkey();
-        let external_spk_21 = external_desc.at_derivation_index(21).script_pubkey();
+        let external_spk_0 = external_desc
+            .at_derivation_index(0)
```

Comment:
Just fyi if you want to keep these all on one line you can use `#[rustfmt::skip]` on the function.